addons/web_tour/static/src/debug/tour_dialog_component.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates id="template" xml:space="preserve">
+
+<t t-name="web_tour.ToursDialog" owl="1">
+    <Dialog title="dialogTitle">
+        <div>
+            <t t-call="web_tour.ToursDialog.Table">
+                <t t-set="caption" t-value="env._t('Onboarding tours')"/>
```

Comment:
I thought we decided to do:
```xml
<t t-set="caption">Onboarding tours</t>
```
Otherwise, the term won't be extracted

Reply:
we did, I lost a diff for some reason